public/faq.html

```diff
@@ -49,6 +49,11 @@
                                 <dt><b>What if I have lost my multi factor device?</b></dt>
                                 <dd>If you have lost the device which is linked to your account for multi factor authentication, then you need to create a TechNow request for your account to be reset. </dd>
                                 <dd>Within the ticket please include that the system is DWP_DataWorks and that you are logging onto the AWS environment. </dd>
+                                <dt><b>Why can't I copy and paste within the analytical environment?</b></dt>
```

Comment:
You can copy and paste within the analytical environment. This FAQ is about pasting between the user's computer and the remote environment. They can only paste between their computer and the remote environment, not copy.